packages/venia-concept/src/components/ProductStickyFooter/productStickyFooterContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
```

Comment:
Can just call this file `container.js` to stay consistent with other components file structure.